db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2018_10_29_081927) do
+ActiveRecord::Schema.define(version: 2018_10_29_154802) do
```

Comment:
cho vào ignore

Reply:
> cho vào ignore

tại sao db/schema.rb phải cho vào .gitignore ạ?

Reply:
vì mỗi lần chạy migrate thì sẽ phải update lại file này mà nội dung thay đổi ko liên quan đến code nên không cần phải đẩy lên github chung của project

Reply:
> vì mỗi lần chạy migrate thì sẽ phải update lại file này mà nội dung thay đổi ko liên quan đến code nên không cần phải đẩy lên github chung của project

noted and fixed